plugins/sqlfluff-templater-dbt/docker/shell

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 my_path="$( cd "$(dirname "$0")"; pwd -P)"
-docker-compose -f ${my_path}/docker-compose.yml exec app bash
+docker compose -f ${my_path}/docker-compose.yml exec app bash
```

Comment:
The `docker-compose` command is deprecated and IIUC has been removed from some recent versions of Docker.